### DISCUSSION TITLE
netlify 환경 변수

### DISCUSSION THREAD

**Discussion by gayeong0916** [team-final/albaform#83] [General]
빌드 오류를 해결하면서 궁금한게 생겨서요.....

정적 생성 함수를 주석 처리 = 클라이언트 호출 -> 빌드 -> 빌드 성공 (현재 pr)
: 클라이언트 호출 = NEXT_PUBLIC_API_BASE_URL 필요
정적 생성 함수 포함 = 서버사이드 호출 -> 빌드 -> 빌드 실패
: 서버사이드 호출 = API_BASE_URL 필요

* NEXT_PUBLIC_API_URL / API_BASE_URL-> 둘 다 netlify 환경 변수에 들어가 있는지가 궁금해요..
* 그게 아니라면 왜 정적 생성 함수만 빌드 했을 때 계속 '유효하지 않은 주소'라고 뜨는지.... 정말정말루 모르겠어욥 ㅠㅠㅠㅠㅠㅠㅠ

**Comment by JiinSeok** [comment]
NEXT_PUBLIC_API_BASE_URL
PRIVATE_API_BASE_URL

저장했습니다...! 잘 몰랐네요 ㅠㅠ
